config/base_boxes.yaml

```diff
@@ -38,10 +38,27 @@ boxes:
     <<: *install_shell
     options: --scenario=katello
 
+  centos6-capsule-nightly:
+    box: centos7
+    ansible:
+      playbook: 'playbooks/capsule.yml'
+      group: 'capsule'
+      server: 'centos6-katello-nightly'
+
   centos7-katello-nightly:
     box: centos7
     <<: *install_shell
-    options: --scenario=katello
+    options: --scenario=katello --disable-selinux
+    ansible:
+      playbook: 'playbooks/dummy.yml'
```

Comment:
If we want just to make it part of other group, we could just skip the playbook part and not set the ansible provisioner